docs/spfx/viva/design/designing-card.md

```diff
@@ -39,7 +39,7 @@ Each dashboard card has the following elements:
 
 When adding the title and heading, ensure the text does not truncate and can fit in the card width on both desktop and mobile in both card sizes (more information about sizes is given below) and all languages used. Truncated text can hide key information that might be necessary for the end user to complete the task. For the title, in most cases, a length of 10-12 characters should fit the card width well.
 
-:::image type="content" source="../../../images/viva-design/img_card_truncation.jpg" alt-text="Card container and elements":::
+:::image type="content" source="../../../images/viva-design/img_card_truncation.jpg" alt-text="Diagram that shows the submit time of... and Time off options, with the Time off option being the recommended option.":::
```

Comment:
Does it make sense to have multiple periods for alt-text.  I think one would suffice.